pkg/ibmcsidriver/node.go

```diff
@@ -379,12 +376,20 @@ func (csiNS *CSINodeServer) NodeGetInfo(ctx context.Context, req *csi.NodeGetInf
 		},
 	}
 
-	// maxVolumesPerNode is the maximum number of volumes attachable to a node; default is 4
-	cores := runtime.NumCPU()
-	if cores >= MinimumCoresWithMaximumAttachableVolumes {
-		maxVolumesPerNode = MaxVolumesPerNode
+	// If environment variable is set, use this value as maxVolumesPerNode
+	value, ok := os.LookupEnv("VOLUME_ATTACHMENT_LIMIT")
+	if !ok {
+		ctxLogger.Warn("Invalid value for VOLUME_ATTACHMENT_LIMIT. Setting the default value:", zap.Reflect("MaxVolumesPerNode", maxVolumesPerNode))
```

Comment:
Please change the log, it is not invalid value, in this case they have not provided the value. You may log - VOLUME_ATTACHMENT_LIMIT is not provided, Setting the default value.

Reply:
thanks, changed the log statement